index.js

```diff
@@ -1,12 +1,12 @@
-/* eslint-disable brace-style, camelcase, semi */
+/* eslint-disable  brace-style, camelcase, semi, no-use-before-define */
```

Comment:
let's actually remove `no-use-before-define` as I don't think we need it

Reply:
I used as I was using Airbnb code styling so it was needed, now removed it as we use standardjs